base/applications/rapps/include/rosui.h

```diff
@@ -97,17 +104,14 @@ class CPointerArray
     BOOL RemoveAt(INT i)
     {
         PVOID ptr = DPA_DeletePtr(m_hDpa, i);
-        if (ptr != NULL)
-        {
-            OnRemoveItem(reinterpret_cast<T*>(ptr));
-            return TRUE;
-        }
-        return FALSE;
+        if (!ptr)
+            return FALSE;
+        return OnRemoveItem(reinterpret_cast<T*>(ptr));
```

Comment:
This makes the code slightly more concise and makes the return value of `RemoveAt` more accurate.